app/models.py

```diff
@@ -34,6 +34,13 @@ def __repr__(self):
     def as_dict(self):
         return {item.name: getattr(self, item.name) for item in self.__table__.columns}
 
+    def get_name(self):
+        """Splits 'firstname.lastname@aun.edu.ng' into a list ['firstname', 'lastname']
+        """
+        name = self.email[:self.email.find('@')]
```

Comment:
In  a case where it is SGA that is the user, how will it be displayed because there is no full stop in their email (that's if I am not mistaken)

Reply:
If it is only one name, the result is a list with only one element: `['firstname']`. But when your are rendering the names on the template you have to check if it's only one name or two so you don't get an error.